### PR TITLE
va-breadcrumb: make current page a link to #content

### PR DESCRIPTION
**Pull request by powellkerry**
## Chromatic

https://2117-breadcrumb-link--60f9b557105290003b387cd5.chromatic.com

## Description
Make the current page text into a link to #content.
- Link should be bold (700) other styles should match the current link style

Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2117

## Testing done
- Local testing in Chrome, Safari, Firefox, and Edge
- Mac VoiceOver

## Screenshots
Before:
Screenshot 2023-10-02 at 2 55 24 PM

After:
Screenshot 2023-10-04 at 3 48 03 PM

## Acceptance criteria
- [ ] Current page link goes to #content
- [ ] Change follows design found [here](https://www.sketch.com/s/a52734dd-00d0-44f1-9c9e-ff4016130e5c/p/F30D6063-F45E-4AAE-A2F5-A26C3E5EA5A5/canvas)
- [ ] Screen readers announce the link as expected

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by jamigibbs** [comment]
The v1 version already has it as an anchor but the hover state seems different from what was done here. Should we align those to be the same? Like the same cursor display?

File: packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx [hunk 1]
```diff
@@ -249,15 +250,17 @@ export class VaBreadcrumbs {
...
           <nav aria-label={label} class={wrapClass}>
-            <ol role="list" onClick={e => this.fireAnalyticsEvent(e)} class="usa-breadcrumb__list">
+            <ol role="list" class="usa-breadcrumb__list">
               {this.formattedBreadcrumbs.map((item, index) => (
                 <li
                   class={`usa-breadcrumb__list-item ${index === this.formattedBreadcrumbs.length - 1 ? 'usa-current' : ''}`}
                   aria-current={index === this.formattedBreadcrumbs.length - 1 ? "page" : undefined}>
                   {index === this.formattedBreadcrumbs.length - 1 ? (
-                    <span>{item.label}</span>
+                    <a class="usa-breadcrumb__link--current" href="#content" onClick={e => this.fireAnalyticsEvent(e)}>
```
[the remark this answers is not in the document]
Comment (powellkerry) [reply packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx]: changed to `va-breadcrumb__link--current`. This is not a USWDS style

**Comment by powellkerry** [comment]
@jamigibbs in conversations with @babsdenney the current page link should match the style of other links including the styles for the hover state, the only difference being that the current page text is bold. I have removed the custom css class that made it different in favor of the following declaration:

```
.usa-current a {
  font-weight: bold;
}
```

**Review by babsdenney — APPROVED** [review]
Design approved!

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**